qa/TL0_multigpu/test_body.sh

```diff
@@ -45,7 +45,7 @@ test_pytorch() {
 }
 
 test_jax() {
-    ${python_new_invoke_test} -s . test_jax_integration_multigpu
+    ${python_new_invoke_test} -s jax test_integration_multigpu
```

Comment:
I moved the tests to a separate folder to make running all JAX related tests easier.